data/account/participationRegistry.go

```diff
@@ -470,7 +517,11 @@ func (db *participationDB) insertInner(record Participation, id ParticipationID)
 		voting := record.Voting.Snapshot()
 		rawVoting = protocol.Encode(&voting)
 	}
-	// PKI TODO: Extract state proof from record.
+
+	// This contains all the state proof data except for the actual secret keys (stored in a different table)
+	if record.StateProofSecrets != nil {
+		rawStateProof = protocol.Encode(&record.StateProofSecrets.SignerContext)
```

Comment:
nit: You could rename `rawStateProof` to `rawStateProofVerifier` if thats more accurate.